e2e_samples/fabric_dataops_sample/README.md

```diff
@@ -64,17 +72,31 @@ The Fabric resources are deployed using the Fabric REST APIs. Once the terraform
     ```bash
     cd ./modern-data-warehouse-dataops/e2e_samples/fabric_dataops_sample/infra
     ```
+1. Create and/or configure your Service Principal [following the Microsoft Fabric Terraform provider instructions for authenticating with Service Principal and Client Secret](https://registry.terraform.io/providers/microsoft/fabric/latest/docs/guides/auth_spn_secret). 
+   1. Make sure your Fabric Admin has enabled the Developer Setting [Service Principals can use Fabric APIs](https://learn.microsoft.com/en-us/fabric/admin/service-admin-portal-developer#service-principals-can-use-fabric-apis) and your Service Principal is allowed to use Fabric APIs.
+   1. Grant your Service Principal Contributor + User Access Administrator permissions on the Azure resource group (see pre-requisites).
+   1. Create a secret and save it for the following step.
+
 
 1. Rename the [.envtemplate](./.envtemplate) file to `.env` and fill in the necessary environment variables. Here is the detailed explanation of the environment variables:
 
     ```bash
-    BASE_NAME="The base name of the Fabric project. This name is used for naming the Azure and Fabric resources."
-    LOCATION="The location of the Azure resources. This location is used for creating the Azure resources."
-    ALDS_GEN2_CONNECTION_ID="The connection ID for the ADLS Gen2 'Cloud Connection'. If not provided, the ALDS Gen2 shortcut creation would be skipped."
+    # Fabric TF provider env variables
+    export FABRIC_TENANT_ID="The Entra ID (Azure AD Tenant Id) of your Fabric tenant"
+    export FABRIC_CLIENT_ID="The Service Principal Client ID (App Id)"
+    export FABRIC_CLIENT_SECRET="The Service Principal Client secret"
+    #
+    # ... (more variables in the file)
     ```
 
     Leave `ALDS_GEN2_CONNECTION_ID` blank for the first run.
 
+1. Import the environment variables file and authenticate to Azure with Service Principal
+    ```bash
+    source .env
+    az login --service-principal -u $FABRIC_CLIENT_ID -p $FABRIC_CLIENT_SECRET --tenant $FABRIC_TENANT_ID
+    ```
+
 1. Review [setup-infra.sh](./infra/setup-infra.sh) script and see if you want to adjust the derived naming of variable names of Azure/Fabric resources.
```

Comment:
The script needs to be reviewed for all the parameters passed. I will add specific comments in the script itself.